fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/responses/RdfStreamStreamingOutput.java

```diff
@@ -116,28 +114,21 @@ private void write(final Iterable<Statement> model,
         final RDFWriter writer = Rio.createWriter(dataFormat, output);
         writer.setWriterConfig(settings);
 
-        for (final Map.Entry<String, String> namespace : excludeProtectedNamespaces(rdfStream.namespaces())) {
-            writer.handleNamespace(namespace.getKey(), namespace.getValue());
-        }
-
-        Rio.write(model, writer);
-    }
-
-    private static Iterable<Map.Entry<String, String>> excludeProtectedNamespaces(
-            final Map<String, String> namespaces) {
         /**
          * We exclude:
          *  - xmlns, which Sesame helpfully serializes, but normal parsers may complain
          *     about in some serializations (e.g. RDF/XML where xmlns:xmlns is forbidden by XML);
          */
-        return Iterables.filter(namespaces.entrySet(), new Predicate<Map.Entry<String, String>>() {
-            @Override
-            public boolean apply(final Map.Entry<String, String> input) {
-                return !input.getKey().equals("xmlns");
-            }
-        });
-    }
+        final List<Map.Entry<String, String>> namespaces = rdfStream.namespaces().entrySet().stream()
+                                                 .filter(x -> !x.getKey().equals("xmlns"))
+                                                 .collect(Collectors.toList());
 
+        for (final Map.Entry<String, String> namespace : namespaces) {
```

Comment:
That why I did [`UncheckedFunction`](https://github.com/fcrepo4/fcrepo4/blob/Java8/fcrepo-kernel/src/main/java/org/fcrepo/kernel/utils/UncheckedFunction.java) and its partners. You may want to reintroduce them.